lib/logging/log.js

```diff
@@ -26,7 +26,7 @@ function NRlog (msg) {
     if (typeof jsMsg.msg !== 'string') {
         jsMsg.msg = JSON.stringify(jsMsg.msg)
     }
-    console.log(`[NR] ${date.toLocaleDateString()} ${date.toLocaleTimeString()} [${jsMsg.level}] ${jsMsg.msg}`)
+    console.log(`[NR] ${date.toLocaleString([], { dateStyle: 'medium', timeStyle: 'medium' })} [${jsMsg.level}] ${jsMsg.msg}`)
```

Comment:
Just noticed this change when running locally. This does *not* match the output in the PR description.

I know this is something you highlighted in https://github.com/flowforge/flowforge/issues/2361 as being inconsistent. I was expecting the device logs to be changed to match Node-RED's, not the otherway around.

Reply:
I think a little context is needed before actioning Nick.

Before this PR, the logging in `log` and `NRLog` were inconsistent in more than just date format - note the `[`square`]` brackets on one but not the other.

#### `log`
'console.log(`[AGENT] ${new Date().toLocaleString([], { dateStyle: 'medium', timeStyle: 'medium' })} ${level || 'info'}:`, msg)'

####  `NRLog`
`console.log(`[NR] ${date.toLocaleDateString()} ${date.toLocaleTimeString()} [${jsMsg.level}] ${jsMsg.msg}`)`

Are you certain you want this reverted?

Reply:
Just noticed in the merge conflict I resolved in [a16b87a](https://github.com/flowforge/flowforge-device-agent/pull/128/commits/a16b87a1e74e6b5192f81ab36752782bddb971ae), you have actually removed the square brackets from the log level - so I will need to update this log entry after all.

There are 2 options: We can revert the change (leaving inconsistent date time formatting) OR remove the square brackets (to align the 2 logging points).

Reply:
Hmm, not sure how those brackets got lost... don't recall editing that line.

My suggestion was to not try to fix the timestamps in this PR as there was a separate issue related to timestamps. However I went ahead and fixed the broader timestamp issue without addressing the inconsistency as we hadn't had this conversation yet....

My preference would be to make them consistent, but the other way around - change the device log to match NR log. It is more succinct and is a consistent length.

Reply:
Resolved in [521e1f6](https://github.com/flowforge/flowforge-device-agent/pull/128/commits/521e1f6df92acfe6d66da1417673a3b717902653) 🤞